manager/manager.go

```diff
@@ -966,6 +966,22 @@ func (m *Manager) becomeLeader(ctx context.Context) {
 				log.G(ctx).WithError(err).Error("failed to create predefined network " + p.Name)
 			}
 		}
+
+		c := store.GetCluster(tx, m.config.SecurityConfig.ClientTLSCreds.Organization())
```

Comment:
The `clusterID` is already accessible to this function, so we can probably use that instead of `m.config.SecurityConfig.ClientTLSCreds.Organization()`.